### PR TITLE
F51-142 & F51-124

### PR DESCRIPTION
**Pull request by robertsoniv**
- Improve product pricing and catalog management views with brief, informative messages.
- Add category count to catalog list views
- Use active dot for category list views
- Link buyer/usergroup category lists back to catalog management category detail view

### CODE REVIEW

File: src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-            Available Catalogs
+            Available Catalogs <br>
+            <small class="text-muted">Select a catalog to manage category level assignments.</small>
         </h3>
-        <!--====== CREATE ======
-        <div class="l-page-header__btns">
-            <button type="button" class="btn btn-primary btn-block" ng-click="buyerCatalogs.createCatalog()"><i class="fa fa-plus-circle"></i>
-                New Catalog
-            </button>
-        </div>-->
```
Comment (crhistianr) [inline src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html]: Can we get rid of this commented out html?

File: src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html [hunk 2]
```diff
@@ -66,6 +61,11 @@ <h3 class="l-page-header__title">
                             <i class="fa fa-caret-down" ng-show="buyerCatalogs.parameters.sortBy == 'Name'"></i>
                             <i class="fa fa-caret-up" ng-show="buyerCatalogs.parameters.sortBy == '!Name'"></i>
                         </th>
+                        <th rowspan="2" ng-click="buyerCatalogs.updateSort('CategoryCount')">
+                            Category Count
+                            <i class="fa fa-caret-down" ng-show="buyerCatalogs.parameters.sortBy == 'CategoryCount'"></i>
+                            <i class="fa fa-caret-up" ng-show="buyerCatalogs.parameters.sortBy == '!CategoryCount'"></i>
+                        </th>
                         <th rowspan="2">
```
Comment (crhistianr) [inline src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html]: I get this message when I try to sort by category count:

{"Errors":[{"ErrorCode":"List.InvalidSortProperty","Message":"Property is not sortable.","Data":{"Property":"Catalog.CategoryCount"}}]}

Doesn't look like its possible

File: src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html (hunk 1 above)
Comment (robertsoniv) [inline src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html]: I believe that diff is telling you I already removed it (it's highlighted in red)

File: src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html (hunk 2 above)
Comment (robertsoniv) [inline src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html]: I will push an update to remove the sorting on that column

**Pull request merged**